include/bout/field3d.hxx

```diff
@@ -525,7 +526,9 @@ public:
 
   bool allowCalcParallelSlices{true};
 
-private:
+  inline Field3DParallel asF3dwy();
```

Comment:
What does `wy` mean?

Reply:
"with y" - I think we should probably rename `asF3dwy()` to `withY()` and `asF3d()` to `dropY()` or `withoutY()`.
Maybe also `withParallel` or something? Happy to discuss different names ...

Reply:
Why not just `asField3Dparallel` and `asField3D`?